pkg/payload/render.go

```diff
@@ -10,6 +10,8 @@ import (
 
 	"github.com/pkg/errors"
 
+	_ "github.com/openshift/api/config/v1/zz_generated.crd-manifests"
```

Comment:
Can you please include a dependencymagnet file like Trevor did in https://github.com/openshift/cluster-version-operator/pull/1045/files (following the [docs](https://github.com/openshift/api/commit/06baaa444cb8bcfd8abb5cf876223ae727c5ec5e#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R211)) instead of pulling it here?